internal/service/alarms/internal/dictionary/dictionary.go

```diff
@@ -116,7 +116,11 @@ var getClientForCluster = k8s_client.NewClientForCluster
 func (r *AlarmDictionary) processCluster(ctx context.Context, version string) ([]monitoringv1.Rule, error) {
 	cluster, err := r.getManagedCluster(ctx, version)
 	if err != nil {
-		return nil, err
+		// TODO remove when we only process cluster version that actually exist
```

Comment:
the error from processCluster is simply logged. It shouldn't stop the server

Reply:
i'll recheck

Reply:
You are correct.  The error is simply logged... not sure why I thought it wasn't.  I'll revert this part of the change.  Thanks.